CHANGELOG.md

```diff
@@ -2,13 +2,15 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/55654104/114202849-e64de080-9957-11eb-8259-8a0879a78740.png)
The call to action selector is a radio button. This means I can't deselect it, but users should be able to select no call to action to their project.

Reply:
Do we need a fix for this now? The user could just go back to the previous step and forward again. I think that just deselecting the radio button when the user clicks it again would be confusing too. I could add a button to clear the radio button but that might mess up the designs.